### PR TITLE
test

### PR DESCRIPTION
**Pull request by VirtuaCreative**

### CODE REVIEW

**Comment by ramosmd** [comment]
wth?

**Comment by VirtuaCreative** [comment]
:oooo:

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 My personal website
 
 For demonstration purposes for now. It's gonna be a real site in a few days! :)
+
+:wave:
```
Comment (ramosmd) [inline README.md:6]: hi

Comment (ramosmd) [reply README.md:6]: aaa

Comment (ramosmd) [reply README.md:6]: `xxxxx`

File: README.md [hunk 2]
```diff
@@ -2,3 +2,5 @@
 My personal website
 
 For demonstration purposes for now. It's gonna be a real site in a few days! :)
```
Comment (ramosmd) [inline README.md:4]: nowwww

Comment (ramosmd) [reply README.md:4]: hwwwww

File: README.md [hunk 3]
```diff
@@ -2,3 +2,5 @@
...
+
```
Comment (ramosmd) [inline README.md:5]: -

**Comment by VirtuaCreative** [comment]
- [x] x
- [x] y

**Comment by VirtuaCreative** [comment]
closes https://github.com/ramosmd/ramosmd.github.io/issues/2

File: README.md (hunk 2 above)
Comment (VirtuaCreative) [inline README.md:4]: jjjjjj

**Pull request merged**